### PR TITLE
fix(multi-env): update the resource path for SPFx project

### PR DESCRIPTION
**Pull request by qinezh**
1. skip to generate bicep template for SPFx project scaffolding.
2. add new manifest template for SPFx, the difference compared to original manifest template:
   * the path of color is updated to `resources/color.png`
   * the path of outline is updated to `resources/outline.png`

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/appstudio/plugin.ts [hunk 1]
```diff
@@ -499,7 +504,7 @@ export class AppStudioPluginImpl {
       );
     }
 
-    const outlineFile = `${appDirectory}/${manifest.icons.outline}`;
+    const outlineFile = `${resourcesDir}/${manifest.icons.outline}`;
```
Comment (a1exwang) [inline packages/fx-core/src/plugins/resource/appstudio/plugin.ts:507]: the outline file path is already the relative path to `resourceDir`, so you cannot prepend `resourceDir` here.
Screenshot from 2021-09-07 16-27-35

Comment (qinezh) [reply packages/fx-core/src/plugins/resource/appstudio/plugin.ts:507]: Nice catch! it affect the azure project indeed. Will fix it soon.

**Review by a1exwang — APPROVED** [review]

File: packages/fx-core/src/core/index.ts [hunk 2]
```diff
@@ -966,5 +969,10 @@ export class FxCore implements Core {
   }
 }
 
+function isAzureProject(solutionSettings: SolutionSettings): boolean {
```
Comment (dooriya) [inline packages/fx-core/src/core/index.ts:972]: This API is also useful in `solution.ts`,  to reduce duplicate code between core & solution, can we move it to some common utility?

Comment (qinezh) [reply packages/fx-core/src/core/index.ts:972]: That would bring a lots of change in `solution.ts`, and make the target of this PR not clear . How about refactoring it in another PR?

Comment (dooriya) [reply packages/fx-core/src/core/index.ts:972]: Sure~ Another PR will be more reasonable.

**Review by dooriya — APPROVED** [review]
:shipit:

**Review by nliu-ms — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**